### ISSUE TITLE
[BUG] Item in final hot bar inventory slot gets removed

### ISSUE THREAD

**Issue by steve4744** [Thatsmusic99/HeadsPlus#9]
## FIRST-HAND CHECK - ARE YOU USING THE LATEST VERSION/SNAPSHOT OF HEADSPLUS? Have you checked to see if others have submitted the same bug report? 
State here when you have checked the following points above.
There are no other open issues.
Present in Version 6.5 and 6.5.1

### Description of the bug
When clicking on item in the heads menu, it removes player's hot bar item in final slot.

### To Reproduce
1. Have an item in slot 9 of the hot bar
2. Run /heads
3. Click on any item
4. Slot 9 item is removed

### Have you been able to reproduce the bug more than once?
Yes

### Expected behavior
Nothing should happen to slot 9

### Has a debug report been generated? 
If you have a debug report, go to plugin/HeadsPlus/debug, copy your report onto pastebin or hastebin, and paste it here, but also skip to the bottom question (additional information). If you don't, fill in the information below.

### Required information
- Server version: 1.14.3
- HeadsPlus version: 6.5.1 (and 6.5)
- Whether or not the plugin uses MySQL - no
- Server type (e.g Spigot, Paper): Spigot
- Uses Vault: Yes
- Any other pieces of information that may be required.

## Additional information
Add any other information/notes you'd like to add at the bottom

I believe this is result of commit 79d4c08

```   
         if (!im.isGlitchSlotFilled() && im.inventory != null) {
                p.getInventory().setItem(8, new ItemStack(Material.AIR));
            }
```
Can't test myself as I'm having trouble getting Eclipse and Maven to play nicely with the new repo and new pom.xml.

**Comment by Thatsmusic99** [comment]
Will investigate now.

**Comment by steve4744** [comment]
Confirmed fixed in 6.5.2. Thanks for doing it so quickly!
